plugins/woocommerce/tests/e2e-pw/global-setup.js

```diff
@@ -57,9 +56,6 @@ module.exports = async ( config ) => {
 	const adminPage = await adminContext.newPage();
 	const customerPage = await customerContext.newPage();
 
-	// Ensure live mode state (coming soon = no)
-	await setOption( request, baseURL, 'woocommerce_coming_soon', 'no' );
```

Comment:
This is already set in the test-env-setup.sh script, so there is no need for it here. More than that, it fails when re-running tests locally.